scrapyd-client/scrapyd-deploy

```diff
@@ -186,8 +187,16 @@ def _get_version(target, opts):
         return str(int(time.time()))
 
 def _upload_egg(target, eggpath, project, version):
-    with open(eggpath, 'rb') as f:
+    if eggpath.startswith('http'):
```

Comment:
This is incorrect. I can call the egg "httpproxy_list_scraper".
Mixing together URLs and paths always causes a mess.